ras_backstage/controllers/collection_exercise_controller.py

```diff
@@ -11,6 +11,22 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
+def get_collection_exercise_by_id(collection_exercise_id):
+    logger.debug('Retrieving collection exercise', collection_exercise_id=collection_exercise_id)
+    url = '{}collectionexercises/{}'.format(app.config["RM_COLLECTION_EXERCISE_SERVICE"],
```

Comment:
f-string or [urljoin](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) would probably be clearer here.